liblangutil/Scanner.cpp

```diff
@@ -317,8 +318,11 @@ Token Scanner::scanSingleLineDocComment()
 	{
 		if (tryScanEndOfLine())
 		{
-			// check if next line is also a documentation comment
-			skipWhitespace();
+			// Check if next line is also a single-line comment.
+			// If any whitespaces were skipped, remember and use source position before.
+			int commentEndPos = m_source->position();
+			endPosition = skipWhitespace() ? commentEndPos : m_source->position();
```

Comment:
I think an `if`-construct would be more readable here.

Reply:
Also ,wouldn't it be better to just set `endPosition` twice instead of using the temporary variable?

Reply:
Ok!